frontend/src/components/broadcast/all_broadcasts/all_broadcasts.js

```diff
@@ -1,6 +1,51 @@
 import React from "react";
 import "./all_broadcasts.css";
+
+import { makeStyles, InputBase, fade } from "@material-ui/core";
+import { Search } from "@material-ui/icons";
+import Dropmenu from "./../../dropmenu/DropMenu";
 import Card from "./card/card";
+
+const useStyles = makeStyles((theme) => ({
+  search: {
```

Comment:
Can we please define these styles in `css` files only

Reply:
every `className` styled there,  uses Mui `themeProvider`.